Improve visusals of sphere by using transparrency

puremourning opened the pull request:
Can you guys check out if this improves the visuals for you. I think this is cleaner and clearer.

puremourning committed:
Improve visusals of sphere by using transparrency

woody1287 committed:
Shinier

Split the ball and sphere materials to allow transparent ball whilst
retaining shiny reflections in sphere.
Also changed shading model for the materials back to unlit (no lights in
scene)

woody1287 commented:
Works nicely for the little ball. Tweaked a touch to reintroduce some of the lighting effects on the large sphere.

puremourning committed:
Only allow one proectile at a time.

The projectile can be refired after leaving the arena (strictly, it is
destroyed)

puremourning committed:
Fix sound when destroying blocks.

Use a different sound when launching the projectile and play sound when a block
is destroyed
